### PR TITLE
Documentation page

### PR DESCRIPTION
**Pull request by Anurag021**
### Added files
   * nav.css
   * side.css
   * docs.css
   
 also copied code from docs.html to index.html, to be hosted in netlify.

### Preview Link
https://enigma-ui.netlify.app/

### CODE REVIEW

**Review by VLeads — APPROVED** [review]
Code is looking good. Go ahead

**Comment by omkarpatke** [comment]
check this spelling mistake
spell-mistake

**Comment by omkarpatke** [comment]
this both links are not working

Screenshot 2022-02-17 123145

File: Components/cards.css [hunk 1]
```diff
@@ -1,9 +1,4 @@
  /* ----- Horizontal Cards----- */
- .card-display{
-    padding: 1rem;
-    border: 1px solid var(--color-gray-200);
-    width: 60%;
-  }
   .horizontal-card-content > *{
     margin:0px;
```
Comment (omkarpatke) [inline Components/cards.css:3]: I think this is not needed

File: Components/sideBar.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
+.side-container{
+    padding: 1rem;
+    width: 20%;
+    position: sticky;
+    top: 0rem;
```
Comment (omkarpatke) [inline Components/sideBar.css]: instead of writing 0rem, you can write only 0 here.

**Review by omkarpatke — CHANGES_REQUESTED** [review]
do some changes as I suggested otherwise all looks good.

**Comment by Anurag021** [comment]
@VLeads Thanks for the review,
@omkarpatke  Thanks for pointing that out, made the required changes can you please approve the changes.

File: Components/cards.css (hunk 1 above)
Comment (Anurag021) [reply Components/cards.css:3]: removing this is taking the card contents outside card.

File: Components/sideBar.css (hunk 2 above)
Comment (Anurag021) [reply Components/sideBar.css]: Thanks, done

**Pull request merged**